br/pkg/aws/ebs.go

```diff
@@ -281,6 +281,155 @@ func (e *EC2Session) DeleteSnapshots(snapIDMap map[string]string) {
 	log.Info("delete snapshot end", zap.Int("need-to-del", len(snapIDMap)), zap.Int32("deleted", deletedCnt.Load()))
 }
 
+// EnableDataFSR enables FSR for data volume snapshots
+func (e *EC2Session) EnableDataFSR(meta *config.EBSBasedBRMeta, targetAZ string) (map[string][]*string, error) {
+	snapshotsIDsMap := fetchTargetSnapshots(meta, targetAZ)
+
+	if len(snapshotsIDsMap) == 0 {
+		return snapshotsIDsMap, errors.Errorf("empty backup meta")
+	}
+
+	eg, _ := errgroup.WithContext(context.Background())
+
+	workerPool := utils.NewWorkerPool(e.concurrency, "enable snapshot FSR")
```

Comment:
I think we don't need to restrict the concurrency here. Perhaps we can just enable all snapshots' FSR and polling the result at a central goroutine by `describe-fast-snapshot-restores` without filters.

Reply:
...If we enable the concurrency, and there are more disks than our concurrency limit, we may cost 2x time for enabling FSR.

Reply:
Enabling/Disabling FSR routines are executed in group of azs. The concurrency is changed to equal to the number of unique azs. So, in reality, the concurrency to enable/disable fsr is 3 or so.

Reply:
If the pool size is always equal to the task, perhaps we can directly use `eg.Go( task )`?